fix white space and section order for cheatsheets

jdorweiler opened the pull request:
This adds two new things:
1, Add in white space for keys `([a]/[b])` so that all of these keys show up as `( [a] / [b] )` on the frontend. 
2. Add missing sections to the end of `section_order`.  This handles case mismatches and spelling.

@moollaza @jagtalon

moollaza commented on share/goodie/cheat_sheets/cheat_sheets.js:
```diff
@@ -34,6 +41,9 @@ DDH.cheat_sheets.build = function(ops) {
         var codeClass = typeof className === "string" ? className : "bg-clr--white";
+
+        // normalize white space for ( [a] / [b] ) keys
+        string = string.replace(/\(\[(.+)\]\/\[(.+)\]\)/, "( [$1] / [$2] )");
```
we actually need to be able to account for multiples of `/ [\.+]` e.g. `( [$1] / [$2] / [$3] )`

jdorweiler commented:
@moollaza  

Updated to handle any multiples of  `[a] / [b] ... ` keys.

Here's how it works.
1. check for `([a]/[b] ..... )`
2. grab the inner portion `[a]/[b]....`
3. split on `/`
4. trim white space for each
4. join with ` / `

GuiltyDolphin commented:
@jdorweiler @moollaza What's the situation with this?

You've got a case of *merge conflicts!*

moollaza commented:
@GuiltyDolphin would you be interested in picking this one up? I think all we need from it would be the normalizing of the content, though even that may not be necessary anymore because devs are doing a better job now...

I think we can close for now though because the new tests enforce devs to make sure their sections are correct

GuiltyDolphin commented:
@moollaza Yeah, I was going to say - the tests should pick up most of this now.
